### PR TITLE
Update approvelist to match new format

### PR DESCRIPTION
**Pull request by JosteinLindhom**
- Combine first and last name to form full name
- Column for approval is now E (from B)

### CODE REVIEW

File: cmd/approvelist/main.go [hunk 1]
```diff
@@ -223,6 +224,17 @@ func fileName(courseCode, suffix string) string {
 }
 
 func loadApproveSheet(courseCode string) (approveMap map[string]int, sheetName string, err error) {
+	// The approve sheet is a single sheet Excel file with five columns:
+	//
+	// 		First name | Last name | Student number    | Candidate number | Approval
+	// 		-----------+-----------+-------------------+------------------+----------
+	// 		<first>    | <last>    | <student_no>      | <candidate_no>   | <approved>
+	//      John       | Doe       | 123456            |                  |
```
Comment (meling) [inline cmd/approvelist/main.go:232]: Indentation is different on this line.

File: cmd/approvelist/main.go [hunk 2]
```diff
@@ -234,8 +246,9 @@ func loadApproveSheet(courseCode string) (approveMap map[string]int, sheetName s
 	sheetName = f.GetSheetName(f.GetActiveSheetIndex())
 	approveMap = make(map[string]int)
 	for i, row := range f.GetRows(sheetName) {
-		if i > 0 && row[0] != "" {
-			approveMap[row[0]] = i + 1
+		if i > 0 && row[0] != "" && row[1] != "" {
```
Comment (meling) [inline cmd/approvelist/main.go:249]: What happens if only one of `row[0]` and `row[1]` is non-empty?

File: cmd/approvelist/main.go [hunk 3]
```diff
@@ -20,10 +20,11 @@ import (
...
 const (
-	srcSuffix = "-original.xlsx"
-	dstSuffix = "-approve-list.xlsx"
-	pass      = "Godkjent"
-	fail      = "Ikke godkjent"
+	srcSuffix      = "-original.xlsx"
+	dstSuffix      = "-approve-list.xlsx"
+	pass           = "Godkjent"
+	fail           = "Ikke godkjent"
+	approvalColumn = "E"
```
Comment (meling) [inline cmd/approvelist/main.go:27]: Would it make sense to instead calculate this by reading the header line?

**Comment by meling** [comment]
Is the reason for this change that they changed the spreadsheet format?

**Comment by JosteinLindhom** [comment]
> Is the reason for this change that they changed the spreadsheet format?

Yes. The current format is as follows:

| Fornavn | Etternavn | Studentnr. | Kandidatnr. | Godkjenning |
|----------|------------|-------------|--------------|---------------|
| Luke | Skywalker | 123456 | | Godkjent |
| Anakin | Skywalker | 654321 | | Ikke godkjent |

The format last year was:

| Student | Godkjenning |
|----------|---------------|
| Luke Skywalker (123456) | Godkjent |
|Anakin Skywalker (654321) | Ikke godkjent |

It would probably make more sense to look at student IDs rather than name with the current format.

File: cmd/approvelist/main.go (hunk 3 above)
Comment (JosteinLindhom) [reply cmd/approvelist/main.go]: Alternate solution provided in 51f3e1d.

**Comment by meling** [comment]
> It would probably make more sense to look at student IDs rather than name with the current format.

Yes, that sounds like a better approach. Did all students add their student IDs?

**Comment by meling** [comment]
Is there more to do here? I think using student IDs is the best solution for matching.

**Pull request merged**